### PR TITLE
feat(travis) update helm repo on teresa releases

### PR DESCRIPTION
**Pull request by yagonobre**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/278)

### CODE REVIEW

File: .helm.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+function download {
+	curl https://storage.googleapis.com/kubernetes-helm/helm-v2.5.1-linux-amd64.tar.gz --output /tmp/helm.tar.gz --silent
+
+	tar -x -C /tmp -f /tmp/helm.tar.gz linux-amd64/helm
+
+	if [[ $TRAVIS_TAG ]]; then
```
Comment (aguerra) [inline .helm.sh]: Missing -z or -n right?

Comment (drgarcia1986) [inline .helm.sh]: `-n` maybe ?

Comment (yagonobre) [inline .helm.sh]: Maybe, but it was working in the two case. Install `aswcli` only if is a tag push

Comment (drgarcia1986) [inline .helm.sh]: @yagonobre what do you think about using `-n` here? (to standardize with [here](https://github.com/luizalabs/teresa/pull/278/files#diff-90f57a2a951e5aa4477cc8c5e3b79904R34) )

Comment (yagonobre) [inline .helm.sh]: updated

**Pull request merged**